Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/Parallel.dfy.expect

```diff
@@ -1,3 +1,4 @@
+Parallel.dfy(267,11): Warning: parentheses around the forall-statement bound variables are deprecated and not needed
```

Comment:
I'd rather remove the parentheses on our test suite. LLMs are going to learn from it :-)
To test the warning message, prefer a dedicated file that includes deprecated in its name and make sure to put as a comment where you expect the warning

Reply:
Okay, I'll remove them from this place, too. (I had kept them in one place in order to test the warning message, but I agree with you that we're better served by not teaching LLMs out there deprecated syntax.)